kwok/charts/crds/karpenter.sh_nodepools.yaml

```diff
@@ -498,6 +498,9 @@ spec:
                       - type
```

Comment:
Should we propose this change as an RFC? There seems to be a bunch of detail around what this status condition means, how we are going to track failures, what it means with respect to scheduling, etc. and I think it would be good to let people see this and get cloudprovder input as well

Reply:
I'm writing the RFC today and I should be able to post tomorrow.